src/tasks/cache/task.rs

```diff
@@ -68,4 +71,25 @@ impl CacheTask {
         let fut = self.task_future(sim_cache);
         (c, tokio::spawn(fut))
     }
+
+    /// Utility to log a bundle and its contents.
+    fn log_bundle(bundle: &TxCacheBundle) {
```

Comment:
per discussion, these events should live in the SDK in `SimCache::ingest`

I also think these are too verbose.